src/AutoVizuA11y.tsx

```diff
@@ -305,7 +346,10 @@ const AutoVizuA11y = ({
 	return (
 		<div
 			ref={ref}
-			onKeyDown={handleKeyDown}
+			onKeyDown={(event) => {
```

Comment:
Can this `alertDiv` query be made inside the `handleKeyDown` function? If we are accessing a ref that means that it is also available inside that callback, right?

Reply:
The alertDiv logic will be changed in a subsequent PR, resolving this issue!